static/js/popovers.js

```diff
@@ -177,6 +177,17 @@ function render_user_info_popover(
     let can_set_away = false;
     let can_revoke_away = false;
 
+    const status_info_values = (exports.status_info_values = {
```

Comment:
I don't think you need to do what you doing in this line. `const status_info_values = {` should be enough, because you are just defining a local variable that is meant to be used only inside this function.